style.css

```diff
@@ -303,3 +306,107 @@ ul {
   margin: 0;
   padding: 0;
 }
+
+.fotter-contact-form-container {
+  min-height: 150vh;
+  border-radius: 100px 0 0;
+  margin-top: 88px;
+  background-color: #6070ff;
+  overflow: hidden;
+  display: flex;
+}
+
+.fotter-contact-form-imag {
+  margin-top: 20px;
+}
+
+.fotter-contact-form-text {
+  margin: 115px -191px 550px 15px;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+}
+
+.fotter-contact-form-heading {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+  margin-bottom: 0;
+  margin-top: 0;
+}
+
+.fotter-contact-form-subheading {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+  margin-bottom: 0;
+  margin-top: 0;
+}
+
+.fotter-contact-form-text form {
+  display: flex;
+  flex-direction: column;
+  align-content: start;
+  justify-content: center;
+  gap: 1em;
+  margin-top: 2em;
+}
+
+.fotter-contact-form-text input {
+  width: 100%;
+  height: 30%;
+  padding: 10px;
+  border-radius: 5px;
+  border: none;
+  outline-style: none;
+}
+
+.fotter-contact-form-text textarea {
+  width: 100%;
+  height: 50%;
+  border-radius: 5px;
+  border: none;
+  outline-style: none;
+  padding: 10px 0 111px 10px;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #b3bac5;
+}
+
+.fotter-contact-form-text input::placeholder,
+.fotter-contact-form-text textarea {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #b3bac5;
+}
+
+.fotter-contact-form-text .btn-footer-container {
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  width: 50%;
+  color: #6070ff;
+}
+
+.fotter-contact-form-text .btn-footer-container:hover {
+  cursor: pointer;
+  background-color: #2f3675;
+  color: #fff;
+}
```

Comment:
- [ ] Ouch! this is close but kindly make sure the button interactions such as, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template below 👇
   ![firefox_7KIb3eS8J1](https://github.com/malodev1234/Portfolio_setup_mobile_first/assets/104968236/12bd6cbf-943d-4c90-a133-d039e432a0d8)